### PR TITLE
Add validation for alias (++)

### PR DESCRIPTION
**Pull request by AlifAlfa73**
Changelog:
- Add validation for valid command that can be alias-ed (Command that have *args as params)
  - Check if command params is 1
  - Check whether that one params is *args
- Refactor ENV file loading (+add override flag to replace existing env var if any with env from file)

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -14,11 +14,12 @@
...
 ENV = os.getenv("ENV")
+env_file = '.env'
 
 if ENV == 'dev':
-    load_dotenv('.env.dev')
-else:
-    load_dotenv('.env')
+    env_file += '.dev'
+
+load_dotenv('.env.dev', override=True)
```
Comment (rusdiy) [inline client.py]: env_file dipake dimana kah gan?

**Review by rusdiy — APPROVED** [review]
sebuah pertanyaan aja gan.
lgtm

File: client.py (hunk 1 above)
Comment (AlifAlfa73) [reply client.py]: laiya mabok WKWK

Comment (AlifAlfa73) [reply client.py]: Updated gan

**Review by rusdiy — APPROVED** [review]

**Pull request merged**